tests/data_context/test_get_data_context.py

```diff
@@ -0,0 +1,109 @@
+"""
+Why does this exist?
+
+"""
+import os
+import pathlib
+from typing import Any, Optional
+from unittest import mock
+
+import pytest
+
+import great_expectations as gx
+from great_expectations.data_context import EphemeralDataContext
+from great_expectations.data_context.types.base import DataContextConfig
+
+
+@pytest.mark.unit
+def test_DataContextOnly(tmp_path):
+    project_path = tmp_path / "empty_data_context"
+    project_path.mkdir()
+    project_path = str(project_path)
+    gx.data_context.DataContext.create(project_path)
+    context_path = os.path.join(project_path, "great_expectations")
+    asset_config_path = os.path.join(context_path, "expectations")
+    os.makedirs(asset_config_path, exist_ok=True)
+    from great_expectations.data_context import DataContext
+
+    os.chdir(project_path)
+    assert isinstance(gx.get_context(), DataContext)
+
+
+# @pytest.mark.parametrize("project_config", "expected", [(DataContextConfig(
+#         config_version=3.0,
+#         plugins_directory=None,
+#         evaluation_parameter_store_name="evaluation_parameter_store",
+#         expectations_store_name="expectations_store",
+#         datasources={},
+#         stores={
+#             "expectations_store": {"class_name": "ExpectationsStore"},
+#             "evaluation_parameter_store": {"class_name": "EvaluationParameterStore"},
+#             "validation_result_store": {"class_name": "ValidationsStore"},
+#         },
+#         validations_store_name="validation_result_store",
+#         data_docs_sites={},
+#         validation_operators={},
+#     ), EphemeralDataContext)])
```

Comment:
This is the parameterization that was giving me trouble... have some trouble passing in the class itself to check `isinstance()`